features/bootstrap/FeatureContext.php

```diff
@@ -85,6 +85,12 @@ public function createDatabase()
         ]);
         $output = new BufferedOutput();
         $application->run($input, $output);
+
+        $input = new ArrayInput([
```

Comment:
Don't create a database for your tests before each scenario, it's horribly slow. Instead create a clean empty database before running your tests, have a `@BeforeScenario` hook starting a transaction and rollback that transaction in `@AfterScenario`. It is way more efficient, the only downside would be if you wanted to take a peek at the database content in the middle of the tests which wouldn't work anymore.

Note that it won't work with SQLite, but I recommend to not use SQLite anyway as it doesn't have real support of any of the RDBSM features of PostgreSQL or MySQL such as transactions, foreign key constraints, cascade persist/delete etc.